README.md

```diff
@@ -1,57 +1,7 @@
-# Northcoders House of Games API
+# Getting started with this repo
 
-## Background
+To use this repo you'll need create 2 new .env files, .env-development and .env-test.data which shouls contain 'PGDATABASE=' with the relevent database name suffixed as per the example in enc-example file.
```

Comment:
- In production, we wouldn't want to reveal db names, however, for the purposes of this readme and project, what benefit might there be to pointing your users to where they might find the db names?

Reply:
I have removed the actual database names from the readme as this file isn't gitignored and instead directed users of the repo to the file where the names can be found